vllm_spyre/v1/worker/spyre_model_runner.py

```diff
@@ -126,6 +146,155 @@ def load_model(self, prompt_lens: Iterable[int],
     def vocab_size(self) -> int:
         return self.model.model.model.config.src_vocab_size
 
+    def _prepare_prompt_cb(
+        self,
+        new_requests: List[NewRequestData],
+    ) -> Tuple[torch.Tensor, torch.Tensor, torch.Tensor, List[int]]:
```

Comment:
Similar to scheduler, I would suggest splitting into two model runner classes `SpyreModelRunnner` and `ContinuousBatchingSpyreModelRunner` or similar